dist/hypermedia.js

```diff
@@ -156,6 +156,20 @@ angular.module('hypermedia')
       }},
 
       /**
+       * Refresh all resources in the context.
+       */
+      refresh: {value: function () {
+        var promises = [];
+        Object.keys(this.resources).forEach(function (key) {
+          var resource = this.resources[key];
+          if (resource.$isSynced) {
```

Comment:
Goed punt. Laten we dan zeggen dat deze methode inderdaad alleen de resources die gesynced waren doet.